plugins/vpp/l2plugin/vppcalls/arp_term_vppcalls_test.go

```diff
@@ -38,7 +38,7 @@ func TestVppAddArpTerminationTableEntry(t *testing.T) {
 			Af: l2ba.ADDRESS_IP4,
 			Un: l2ba.AddressUnion{Union_data: [16]byte{192, 168, 4, 4}},
 		},
-		Mac: l2ba.MacAddress{Bytes: []byte{0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF}},
+		Mac: l2ba.MacAddress([6]uint8{255, 255, 255, 255, 255, 255}),
```

Comment:
The extra conversion is unnecessary. Just initialize directly like this: `l2ba.MacAddress{255, 255, ...}`.